install.sh

```diff
@@ -159,10 +164,24 @@ install_packages( )
 #     yum -y update brings *all* installed packages up to date
 #     without seeking user approval
 #     elevate_if_not_root yum -y update
-      install_yum_packages "${library_dependencies_centos[@]}"
-
-      if [[ "${build_clients}" == true ]]; then
-        install_yum_packages "${client_dependencies_centos[@]}"
+      if [[ ( "${VERSION_ID}" -ge 8 ) ]]; then
+        install_yum_packages "${library_dependencies_centos_8[@]}"
+
+        if [[ "${build_clients}" == true ]]; then
+          install_yum_packages "${client_dependencies_centos_8[@]}"
+        fi
+      elif [[ ( "${VERSION_ID}" -ge 6 ) ]]; then
+        install_yum_packages "${library_dependencies_centos_6[@]}"
+
+        if [[ "${build_clients}" == true ]]; then
+          install_yum_packages "${client_dependencies_centos_6[@]}"
+        fi
```

Comment:
7 will fall into this case, >= is not what you want just -eq , no?

Reply:
Yep, you're definitely right about that, stupid mistake on my part. I've fixed it in my next commit